src/app/[lang]/onboarding/introduce-yourself/introduce.tsx

```diff
@@ -24,27 +27,35 @@ export default function Introduce({
     }
   };
 
+  const handleSubmit = async (e: any) => {
+    e.preventDefault();
+    await IntroducePost(introduce, lang);
+  };
+
   return (
     <div className="mt-10">
-      <textarea
-        value={introduce}
-        max-length="300"
-        onChange={handleText}
-        className="inline-flex h-[192px] w-full items-center justify-center rounded-3xl border border-neutral-700 bg-stone-900 px-5 py-3 font-main text-sm font-normal leading-tight text-neutral-50 outline-none focus:border-violet-700"
-        name="introduce"
-        id="introduce"
-        placeholder={dict.typeHere}
-      ></textarea>
-      <div className="-mt-2 px-2 text-right">
-        <span className="font-main text-xs font-normal leading-none text-stone-300">
-          {textLength} / 300
-        </span>
-      </div>
-      <button className="main__btn mt-24">
-        <Link className="main__link" href="">
-          {dict.nextStep}
-        </Link>
-      </button>
+      <form>
+        <textarea
+          value={introduce}
+          max-length="300"
+          onChange={handleText}
+          className="inline-flex h-[192px] w-full items-center justify-center rounded-3xl border border-neutral-700 bg-stone-900 px-5 py-3 font-main text-sm font-normal leading-tight text-neutral-50 outline-none focus:border-violet-700"
+          name="introduce"
+          id="introduce"
+          placeholder={dict.typeHere}
+        ></textarea>
+        <div className="-mt-2 px-2 text-right">
+          <span className="font-main text-xs font-normal leading-none text-stone-300">
+            {textLength} / 300
+          </span>
+        </div>
+        <button onClick={handleSubmit} className="main__btn mt-24">
+          <Link className="main__link" href="">
+            {dict.nextStep}
+          </Link>
+        </button>
```

Comment:
The link don't need to be used here

```suggestion
        <button onClick={handleSubmit} className="main__btn main__link mt-24">
            {dict.nextStep}
        </button>
```